tracetree.h

```diff
@@ -76,7 +76,7 @@ class TraceTree
             if (it == parent->children.end() || it->instructionPointer != ip) {
                 index = m_index++;
                 it = parent->children.insert(it, {ip, index, {}});
-                fprintf(out, "t %" PRIxPTR " %x\n", reinterpret_cast<uintptr_t>(ip), parent->index);
```

Comment:
Please note that I'm depending on C++11, and I'm perfectly fine with depending on C99. If a platform does not support PRIxPTR then heaptrack will probably fail elsewhere:

- I'm using uintptr_t extensively
- I'm even assuming that uintptr_t will always fit into a uint64_t when analyzing the data

So, again: If you want me to merge your pull request, please take out this hunk. Adding the cinttypes include is a good change by itself and I would like to have this in, now. Otherwise I'll do it myself.

Thanks a lot